react-rpg/src/Pages/Store/Store.jsx

```diff
@@ -1,5 +1,46 @@
+import { useState } from 'react';
+import Header from "../../Components/Header/Header";
+import CardGrid from '../../Components/CardGrid/CardGrid';
+import CharacterSheet from '../Character/CharacterSheet/CharacterSheet';
+import data from '../Store/_mocks/items.json'
+import Inventory from '../Inventory/Inventory';
+
+
 const Store = () => {
-   return <h1>Store</h1>
+   
+   const [items, setItems] = useState(data);
+
+   const [selectedCharacter, setSelectedCharacter] = useState(null);
+   const onSelectCharacter = (name) => { setSelectedCharacter(name)};
+
+   const [inventory, setInventory] = useState([]);
+   const buyItem = (itemName) => {
+      setInventory((prevItems) => [...prevItems, itemName]);
+      setItems((prevItems) => prevItems.filter((currItem) => currItem.name !== itemName ));
+      console.log(items);
+    };
+
+   
+
+   let quote = 
```

Comment:
This would be better stored as part of the Character's data, so make a character an object that has a name, a quote, and some items or something